docs/dictionary/command/send.lcdoc

```diff
@@ -4,7 +4,7 @@ Type: command
 
 Syntax: send <message> [ to <object> [in <time> [seconds | ticks | milliseconds]] ]
 
-Summary: Sends a <message> to an <object>.
+Summary: Either sends <message> to <object> immediately (if "in" is not specified), or adds <message> to the <pendingMessages><function> queue to be sent to <object> at a later time (if "in" is specified).
```

Comment:
The modified summary is too long here, and you should revert your removal of 'a' and 'an' - perhaps change it to
```
Either sends a <message> to an <object> immediately, or adds a <message> to the message queue.
```